lib/Fmt_ast.ml

```diff
@@ -4344,9 +4352,17 @@ and fmt_let c ~ext ~rec_flag ~bindings ~parens ~fmt_atrs ~fmt_expr ~body_loc
        $ hvbox 0 fmt_expr ) )
   $ fmt_atrs
 
-and fmt_value_binding c ~rec_flag ?ext ?in_ ?epi
+and fmt_value_binding c ~rec_flag ?ext ?in_ ?loc_in ?epi
     {lb_op; lb_pat; lb_args; lb_typ; lb_exp; lb_attrs; lb_loc; lb_pun} =
-  update_config_maybe_disabled c lb_loc lb_attrs
+  let loc_with_in =
+    match loc_in with
+    | None -> lb_loc
+    | Some loc_in ->
+        { loc_start= lb_loc.loc_start
+        ; loc_end= loc_in.loc_end
+        ; loc_ghost= lb_loc.loc_ghost || loc_in.loc_ghost }
+  in
+  update_config_maybe_disabled c loc_with_in lb_attrs
```

Comment:
Another approach would be to still print `in_` and `epi` when disabled

Reply:
With the current approach, could the `in_` be dropped if `loc_in` is `None` ? I would be more comfortable with a new function that takes a non-optional `loc_in`.

Reply:
This is addressed in the recent commit :tada: